tests/simple_sum.asm

```diff
@@ -0,0 +1,129 @@
+// Verfies that a sum in the input has been computed properly.
+// Input: sum, cnt, x_1, x_2, ..., x_cnt
+// This input is assumed to be present in a minirust variable called "input"
+// of type "Vec<FieldElement>"
+
+// Code in `${`...`}` is rust-like code that is run by the prover
+// to generate free inputs.
+
+reg A;
```

Comment:
Had to require `;` because the pil lexer and the asm lexer need to be the same since you can include pil inside asm and the pil lexer ignores all whitespace.